governance/charter.md

```diff
@@ -0,0 +1,102 @@
+# SIG-Security Charter (draft)
+
+This charter describes operations as a [CNCF SIG](https://github.com/cncf/toc/blob/master/sigs/). The [Focus](#focus) section below describes what is in and out of scope,
+and [Governance](#governance) section describes how our operations are consistent with CNCF policies with links to more detailed documents.
+
+**Mission:** to reduce risk that cloud native
+applications expose end user data or allow other unintended access.
```

Comment:
suggestion:

I feel like unauthorized <--> unintended may be a more resonant word with risk/security.

"To reduce the risk of unauthorized access and end user data exposure of cloud native applications"

Reply:
seems reasonable to me -- since this was moved from original charter in README, will wait for other folks to review and chime in.

Reply:
+1

Reply:
changed to unauthorized -- thanks @lumjjb and @lizrice